release-notes/major11/11.0.0/valtimo-backend-libraries.md

```diff
@@ -31,7 +31,13 @@ The following bugs were fixed:
 
 The following breaking changes were introduced:
 
-* **Breaking change1**
+* **Removed REST endpoints**
+  Several endpoints were meant for public use. By default, Valtimo should not offer any public endpoints,
+  and as a result these have been moved. This is the list of removed endpoints:
```

Comment:
> these have been moved
To where? I believed they were `removed`.

Reply:
Moved to the garbage bin! You're correct though, fixed it